tools/pythonpkg/tests/fast/test_alex_multithread.py

```diff
@@ -61,8 +61,9 @@ def test_same_connection(self, duckdb_cursor):
 
         assert duckdb_con.execute("""SELECT * FROM my_inserts order by thread_name""").fetchall() == [('my_thread_0',), ('my_thread_1',), ('my_thread_2',)]
 
-    def test_multiple_cursors_persisted(self, duckdb_cursor):
-        duckdb_con = duckdb.connect('another_test_10.duckdb')
+    def test_multiple_cursors_persisted(self, request):
+        dbname = f'{request.node.name}.duckdb'
```

Comment:
After I wrote this code I realised we should probably use the `tmp_path` fixture to avoid race conditions in parallel test execution instead